live-ingester/src/integration-test/resources/rest/hxinsight/requests/create-or-update-document.yml

```diff
@@ -35,16 +36,19 @@ body: [
         "annotation": "dateModified"
       },
       "cm:versionType": {
+        "type": "string",
         "value": "MAJOR"
       },
       "cm:versionLabel": {
+        "type": "string",
         "value": "1.0"
       },
       "createdBy": {
         "value": "admin",
         "annotation": "createdBy"
       },
       "ALLOW_ACCESS": {
+        "type": "object",
```

Comment:
Arrays of strings should be `string`. The empty array for `DENY_ACCESS` should also be `string`, but it's not easy to determine this. My suggestion is to strip all empty arrays from the ingest message for the time being.